generator/src/main/java/org/corfudb/generator/distributions/Streams.java

```diff
@@ -0,0 +1,36 @@
+package org.corfudb.generator.distributions;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * This class implements a distribution over the possible streams that
+ * can be created/modified.
+ *
+ * Created by maithem on 7/14/17.
+ */
+public class Streams extends DataSet {
+
+    final private Set<UUID> streams;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename field "streams" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)